src/components/Step.vue

```diff
@@ -289,7 +294,7 @@ export default class Step extends Vue {
   cursor: move;
 }
 
-.query-pipeline-step__action i {
+.query-pipeline-step__action ::v-deep .fa-cog {
```

Comment:
Same here, you may be able ot use a class in the FAIcon component instead

Reply:
As said in description, will handle all the v-deep .fa logic in another PR, this one is too long already